src/Modules/ArticleById.jsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from "react"
+import {useParams} from 'react-router-dom'
+import BigCard from "./BigCard"
+import "../App.css"
+
+export default function ArticleById() { 
+    const {article_id} = useParams()
+
+    let [article, setArticle] = useState([])
```

Comment:
Why the use of let when destructuring useState here? Const woud be preferable, because we use the set state function (setArticle) to change the state value (article) opening up further room for reassignment in our component doesn't seem necessary/could open up room for errors